build/rewrite.go

```diff
@@ -54,12 +54,59 @@ func allowedSort(name string) bool {
 	return false
 }
 
-// Rewrite applies the high-level Buildifier rewrites to f, modifying it in place.
+// RewristeSet will contain rewrites to be applied.
+type RewriteSet struct {
+	Names []string
+}
+
+// Rewrites controls the rewrites to be applied.
+//
+// If non-nil, the rewrites with the specified names will be run. If
+// nil, a default set of rewrites will be used that is determined by
+// the type (BUILD vs default starlark) of the file being rewritten.
+type Rewriter struct {
+	Rewrites                        *RewriteSet
+	LabelArg                        map[string]bool
+	Denylist                        map[string]bool
+	SortableListArg                 map[string]bool
+	SortDenyList                    map[string]bool
+	SortAllowList                   map[string]bool
+	NamePriority                    map[string]int
+	StripLabelLeadingSlashes        bool
+	ShortenAbsoluteLabelsToRelative bool
```

Comment:
I only see the usage of `Rewrites` and `NamePriority` in the code, do the other attributes affect anything?